CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ endif()
 macro(build_benchmark)
   set(extra_cmake_args)
 
-  set(GOOGLE_BENCHMARK_TARGET_VERSION "1.6.1")
+  set(GOOGLE_BENCHMARK_TARGET_VERSION "1.8.0")
```

Comment:
As with other vendored packages, 1.6.1 is the version currently released into Ubuntu 22.04.  So I'd prefer to stay with that version unless there is a strong justification to moving to a newer version.